wp-parsely.php

```diff
@@ -28,18 +30,11 @@
  * Keep track of Parse.ly loading info in one place.
  */
 final class Parsely_Loader_Info {
-	// Describes how the parse.ly plugin is integrated/loaded.
-	const INTEGRATION_TYPE_MUPLUGINS        = 'MUPLUGINS';
-	const INTEGRATION_TYPE_MUPLUGINS_SILENT = 'MUPLUGINS_SILENT';
-	const INTEGRATION_TYPE_SELF_MANAGED     = 'SELF_MANAGED';
-	const INTEGRATION_TYPE_NONE             = 'NONE';
```

Comment:
- Added `Parsely_Integration_Type` class which act as enum and moved all of these constants there.
- Apart from this also updated const values which are now more clear.